rosdep/python.yaml

```diff
@@ -1345,6 +1345,9 @@ python-docx:
   fedora: [python-docx]
   ubuntu:
     pip: [python-docx]
+python-dt-apriltags-pip:
+  ubuntu:
```

Comment:
could you include also debian, fedora, opensuse, osx?

do you want a python 2 entry or a python 3 one?
For the second, you need to add a `python3-dt-apriltags-pip` rule instead.